app/controllers/instrument_issues_controller.rb

```diff
@@ -37,9 +43,29 @@ def init_product
   end
 
   def init_instrument_issue
-    @instrument_issue = InstrumentIssue.new(product: @product, 
+    @instrument_issue = InstrumentIssue.new(product: @product,
                                             user: current_user,
                                             order_detail: @order_detail)
   end
 
+  def redirect_to_path
+    redirect_to_order_id = params[:redirect_to_order_id]
+
+    if redirect_to_order_id.present?
+      order = Order.find(redirect_to_order_id)
+
+      if order.present?
+        facility_order_path(order.facility, order)
+      else
+        reservations_path
```

Comment:
do we expect to be hitting this?  maybe add a comment to say when we would hit it, or that we're just putting it in for safety